docs/user/alerting/action-types/pre-configured-connectors.asciidoc

```diff
@@ -64,3 +64,27 @@ Clicking a preconfigured connector shows the description, but not the configurat
 
 [role="screenshot"]
 image::images/pre-configured-connectors-view-screen.png[Pre-configured connector view details]
+
+[float]
+[[preconfigured-connector-alert-history]]
+==== Alert History Preconfigured Connector
+
+{kib} offers a preconfigured <<index-action-type, Index>> connector to facilitate indexing active alert data into {es}.
+
+To use this connector, set the <<action-settings, `xpack.actions.preconfiguredAlertHistoryEsIndex`>> configuration to `true`. 
+
+```js
+  xpack.actions.preconfiguredAlertHistoryEsIndex: true
+```
+
+When creating a new rule, add an <<index-action-type, Index action>> and select the `Alert History ES Index (preconfigured)` connector.
```

Comment:
This link confuses me.  It takes me to the Index connector and action page.  I'm still not sure what the actions are and how it relates to the screenshot.

Reply:
This is a type of index connector, which is why I added the link to the index connector type. Normally with an index connector, the user who configures the connector decides what the index name is and the user who adds the connector to a rule decides what the data schema is. For this preconfigured alert history connector, the index name and data schema is pre-determined.